project.clj

```diff
@@ -1,22 +1,24 @@
-(defproject crypto-password "0.2.0"
+(defproject crypto-password "0.2.1"
   :description "Library for securely hashing passwords"
   :url "https://github.com/weavejester/crypto-password"
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]
+  :dependencies [[org.clojure/clojure "1.10.0"]
                  [crypto-random "1.2.0"]
                  [crypto-equality "1.0.0"]
-                 [commons-codec "1.10"]
-                 [org.mindrot/jbcrypt "0.3m"]
+                 [commons-codec "1.12"]
+                 [at.favre.lib/bcrypt "0.7.0"]
                  [com.lambdaworks/scrypt "1.4.0"]]
   :plugins [[lein-codox "0.9.4"]]
   :codox
   {:output-path "codox"
    :project  {:name "Crypto-Password"}
    :metadata {:doc/format :markdown}
    :source-uri "http://github.com/weavejester/crypto-password/blob/{version}/{filepath}#L{line}"}
-  :aliases {"test-all" ["with-profile" "+1.6:+1.7:+1.8" "test"]}
+  :aliases {"test-all" ["with-profile" "+1.6:+1.7:+1.8:+1.9:+1.10" "test"]}
```

Comment:
Adding more test profiles is a good change, but needs to be in a separate commit and PR.